airflow/providers/slack/transfers/sql_to_slack.py

```diff
@@ -58,6 +62,7 @@ class SqlToSlackApiFileOperator(BaseSqlToSlackOperator):
     :param slack_base_url: A string representing the Slack API base URL. Optional
     :param slack_method_version: The version of the Slack SDK Client method to be used, either "v1" or "v2".
     :param df_kwargs: Keyword arguments forwarded to ``pandas.DataFrame.to_{format}()`` method.
+    :param allow_null: Keyword argument to allow null sql output which implies allowing sending an empty file.
```

Comment:
```suggestion
    :param send_file_if_empty: Keyword argument to allow null sql output which implies allowing sending an empty file.
```

I think it's easier to understand